code/game/objects/machinery/Sleeper.dm

```diff
@@ -445,16 +445,18 @@
 		beaker.loc = usr.loc
 		beaker = null
 
+/obj/machinery/sleeper/MouseDrop_T(mob/M, mob/user)
+	move_inside(M)
 
-/obj/machinery/sleeper/verb/move_inside()
+/obj/machinery/sleeper/verb/move_inside(mob/M)
```

Comment:
Having a verb have an argument will make it open up a prompt, did you test this?

Reply:
This will only ask when you right click the machine and select move inside.
A better solution is make a proc called like do_move_inside that your drag over code and move_inside both call